dockstore-webservice/src/main/resources/openapi3/openapi.yaml

```diff
@@ -6487,6 +6487,31 @@ paths:
       tags:
       - workflows
   /workflows/{workflowId}/workflowVersions:
+    get:
+      description: Return first 200 versions in an entry
+      operationId: getWorkflowVersions
+      parameters:
+      - description: id of the worflow
+        in: path
+        name: workflowId
+        required: true
+        schema:
+          type: integer
+          format: int64
+      responses:
+        default:
```

Comment:
missing response, could probably leave out swagger

Reply:
added response, it's easier for me if the swagger stays for the tests. But I can change it if you feel strongly about it.

Reply:
ok, just use openapi as much as possible then. missing response codes